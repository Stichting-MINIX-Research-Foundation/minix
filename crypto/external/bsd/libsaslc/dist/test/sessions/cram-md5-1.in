CRAM-MD5
2
AUTHID Ali Baba
PASSWD Open, Sesame
<68451038525716401353.0@localhost>
