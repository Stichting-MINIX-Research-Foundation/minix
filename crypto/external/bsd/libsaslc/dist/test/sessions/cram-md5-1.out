Ali Baba 6fa32b6e768f073132588e3418e00f71
